Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -145,6 +145,8 @@ def parse_args(args):
             help="Run a concurrent JavaScript thread with each test"),
         optparse.make_option("--dump-render-tree", "-1", action="store_false", default=True, dest="webkit_test_runner",
             help="Use DumpRenderTree rather than WebKitTestRunner. This runs the wk1 single-process architecture."),
+        optparse.make_option("--webkit-test-runner", "-2", action="store_true", default=True, dest="webkit_test_runner",
+            help="Use WebKitTestRunner exclusively, skip any wk1 specific tests."),
```

Comment:
How does it work for both to write into the same variable, and to have a default value? Will default value for `--webkit-test-runner` override the explicit option for `--dump-render-tree`?

Reply:
I think that this different version is fragile and unnecessarily convoluted, as it duplicates pasting behavior. Was there some obstacle when doing it in the way that I suggested?

Reply:
I was trying to avoid using two variables, that's going to end up being a larger change, things we're sort of planning on doing eventually anyways, but my hope was that this would be a quick change...apparently not